PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera.rbxmx

```diff
@@ -18,11 +18,6 @@ if not VRServiceExists or not VRService then
 	VRService = UserInputService
 end
 
-local spawnLocationFixFlagExists, spawnLocationFixFlagEnabled = pcall(function()
-	return settings():GetFFlag("SpawnLocationFixEnabled")
-end)
-local FFlagSpawnLocationFixEnabled = spawnLocationFixFlagExists and spawnLocationFixFlagEnabled
```

Comment:
This was merged in because I am bad at git. The actual flag query is lower in the file. We used UserSettings() instead